src/awesome_map/BoundaryInterceptor.js

```diff
@@ -150,6 +154,20 @@ define(function(require) {
          */
         this._easing = options.easing || EasingFunctions.easeOutQuart;
 
+        this._boundarySensitivity = options.boundarySensitivity || DEFAULT_BOUNDARY_SENSITIVITY;
+        /**
+         * Observable for subscribing to scroll events beyond AwesomeMap boundaries.
+         *
+         * @method AwesomeMap#onScrollPastTopBoundary
+         * @param {Function} callback
+         *        Invoked with (sender, {
+         *            boundary: {@link BoundaryTypes}
+         *        })
+         */
+        // @TODO: Does this need to be disposed somewhere?  If so, where?
```

Comment:
That's what i was thinking, but i didnt want to make any assumptions, as memory leaks are icky.  But i cant really see any case where the AwesomeMap would be disposed during the life of a page instance, let alone disposed enough times to cause a memory issue even if one observable wasnt getting cleaned up properly.